src/browser/styles/bootstrap.grid-only.min.css

```diff
@@ -7,8 +7,1100 @@
+ */ /*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Yeah, this seems related to [this comment](https://github.com/neo4j/neo4j-browser/pull/1014/#discussion_r355902452). It probably wasn't meant to be formatted.

Reply:
👍